### PR TITLE
13553 Edit UI: BEN IA correction should prompt for Completing Party

### PR DESCRIPTION
**Pull request by lewischenstudio**
*Issue #:* /bcgov/entity#13553

*Description of changes:*
- add Completing Party component (for Client Error only)
    - place above Detail component
- unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/views/Correction/BenCorrection.vue [hunk 1]
```diff
@@ -27,6 +27,10 @@
 
     <Articles class="mt-10" />
 
+    <template v-if="isClientErrorCorrection">
+      <CompletingParty class="mt-10" sectionNumber="1." validate="true" />
```
[the remark this answers is not in the document]
Comment (lewischenstudio) [reply src/views/Correction/BenCorrection.vue:31]: Thanks. It's now fixed. Same as in `FmCorrection`.

**Review by lambert-alex — APPROVED** [review]
LGTM

File: src/views/Correction/FmCorrection.vue [hunk 2]
```diff
@@ -28,7 +28,7 @@
 
     <StaffPayment
       class="mt-10"
-      :sectionNumber="isClientErrorCorrection ? '4.' : '2.'"
+      :sectionNumber="isClientErrorCorrection ? '4.' : '3.'"
```
Comment (severinbeauvais) [inline src/views/Correction/FmCorrection.vue:31]: I think the previous code was actually correct: if this is not a client error correction then both Completing Party and Certify Section are not rendered, so StaffPayment is indeed the second section.

Comment (severinbeauvais) [reply src/views/Correction/FmCorrection.vue:31]: I also wondering why the Certify Section is not displayed here but it IS displayed in a BEN correction? Please confirm with Mihai.

Comment (lewischenstudio) [reply src/views/Correction/FmCorrection.vue:31]: You are right. Certify will be displayed for client error, so StaffPayment is the second section. I will change it back in my this PR: https://github.com/bcgov/business-edit-ui/pull/380. I will confirm with Mihai during today's standup and add the condition to Certify if we want to hide Certify for staff error for BEN IA correction.

**Pull request merged**